src/dstack/_internal/core/backends/oci/compute.py

```diff
@@ -197,3 +200,22 @@ def _supported_instances(offer: InstanceOffer) -> bool:
     if "Flex" in offer.instance.name:
         return False
     return any(map(offer.instance.name.startswith, SUPPORTED_SHAPE_FAMILIES))
+
+
+def _get_availability_domain_or_error(
+    availabile_domains: List[str],
```

Comment:
(nit) Same typo in a few places
```suggestion
    available_domains: List[str],
```